http/oidc/src/main/java/org/wildfly/security/http/oidc/TokenValidator.java

```diff
@@ -213,11 +224,24 @@ public TokenValidator build() throws IllegalArgumentException {
             jwtConsumerBuilder = new JwtConsumerBuilder()
                     .setExpectedIssuer(expectedIssuer)
                     .setJwsAlgorithmConstraints(
-                            new AlgorithmConstraints(AlgorithmConstraints.ConstraintType.PERMIT, expectedJwsAlgorithm))
-                    .setRequireExpirationTime();
+                            new AlgorithmConstraints(AlgorithmConstraints.ConstraintType.PERMIT, expectedJwsAlgorithm));
+
+            if (!skipExpirationValidator) {
+                jwtConsumerBuilder.setRequireExpirationTime();
+            }
 
             return new TokenValidator(this);
         }
+
+        public Builder setSkipExpirationValidator() {
+            this.skipExpirationValidator = true;
+            return this;
+        }
+
+        public Builder setTokenType(String tokenType) {
```

Comment:
Just wondering if this method is needed?

Reply:
Yes, it's used in LogoutHandler. The default value, Bearer, for tokenType must be overridden.